src/RewardsStreamerMP.sol

```diff
@@ -99,6 +100,40 @@ contract RewardsStreamerMP is IStakeManager, TrustedCodehashAccess, ReentrancyGu
         account.lastMPUpdateTime = block.timestamp;
     }
 
+    function lock(uint256 lockPeriod) external onlyTrustedCodehash nonReentrant {
+        if (lockPeriod == 0 || (lockPeriod < MIN_LOCKUP_PERIOD || lockPeriod > MAX_LOCKUP_PERIOD)) {
```

Comment:
I this we just need:

```suggestion
        if (lockPeriod < MIN_LOCKUP_PERIOD || lockPeriod > MAX_LOCKUP_PERIOD) {
```
in this case `0`  is already inside `lockPeriod < MIN_LOCKUP_PERIOD`